skt/executable.py

```diff
@@ -180,7 +190,12 @@ def cmd_merge(cfg):
         cfg:    A dictionary of skt configuration.
     """
     global retcode
-    idx = 0
+    # Counter merge patch following:
+    # id[0]:  counter of merge-ref option.
+    # idx[1]: counter of patch option.
+    # idx[2]: counter of pw option.
+    idx = [0, 0, 0]
+    previous_merge = ['', '']
```

Comment:
Why use a list of two strings here when only the first one (the name of the option) is used in the comparisons and the second one is never touched? Wouldn't it be easier to use a single string, to which the name of the applied option is assigned instead?

Reply:
Right. It is redundant